R/dt_options.R

```diff
@@ -19,72 +19,96 @@ dt_options_init <- function(data) {
     "container_height",                  FALSE,  "container",        "px",      "auto",
     "container_overflow_x",              FALSE,  "container",        "overflow","auto",
     "container_overflow_y",              FALSE,  "container",        "overflow","auto",
-    "font_color",                         TRUE,  "table",            "value",   "#333333",
-    "font_color_light",                   TRUE,  "table",            "value",   "#FFFFFF",
     "table_id",                          FALSE,  "table",            "value",   NA_character_,
     "table_width",                        TRUE,  "table",            "px",      "auto",
-    "table_font_size",                    TRUE,  "table",            "px",      "16px",
-    "table_background_color",             TRUE,  "table",            "value",   "#FFFFFF",
     "table_margin_left",                  TRUE,  "table",            "px",      "auto",
     "table_margin_right",                 TRUE,  "table",            "px",      "auto",
+    "table_background_color",             TRUE,  "table",            "value",   "#FFFFFF",
+    "table_font_color",                   TRUE,  "table",            "value",   "#333333",
+    "table_font_color_light",             TRUE,  "table",            "value",   "#FFFFFF",
+    "table_font_size",                    TRUE,  "table",            "px",      "16px",
```

Comment:
*Not something to be fixed in this PR,* but it feels a little arbitrary which parts of the table get to change which font attributes. Also I might quibble (or bikeshed!) with which options can be set, e.g. you can't set different amounts of vertical and horizontal padding.

Reply:
Let's talk more about this. I'd love to have all possible options for HTML table to be settable. I don't really think I have a great strategy to make this easy (and not overwhelming). I basically made some editorial decisions here but they are highly opinionated (and yes, arbitrary).